prettier/main.ts

```diff
@@ -139,15 +140,20 @@ async function formatFile(
     return;
   }
 
-  const formatted = prettier.format(text, {
+  const formatted: string = prettier.format(text, {
     ...prettierOpts,
     parser,
     plugins: prettierPlugins
   });
 
   if (text !== formatted) {
```

Comment:
There is a question here that needs to be discussed.

If format a formatted file, do we still need to output to `stdout`?

Reply:
The original prettier cli seems working like that (output always to stdout when `--write` is not specified). So I think it's better to follow it.

Reply:
This may still have some problems, give me some time to confirm.